otherlibs/stdune/src/io.ml

```diff
@@ -179,6 +179,7 @@ module Copyfile = struct
   let sendfile_with_fallback ?chmod ~src ~dst () =
     try sendfile ?chmod ~src ~dst ()
     with Unix.Unix_error (EINVAL, "sendfile", _) ->
+      Unix.unlink dst;
```

Comment:
Is this the correct place to do this, or can it be done inside `sendfile` when it aborts?

Reply:
I think there is some similar cleanup for src there too.

Reply:
I thought as well, but this happens in the prepare phase, not when `sendfile` itself fails.